examples/json-arrays/json_arrays.description

```diff
@@ -1,2 +1,2 @@
 // JSON array literals are written exactly the same way as Ballerina arrays.
-// JSON array values must be of the JSON types: string, number, object, array, boolean, or null.
+// JSON array values must be of the `string`, `number`, `object`, `array`, `boolean`, or `null` JSON types.
```

Comment:
We don't need to emphasize the types here since they are not used as references to the types in this context. Some of the aren't keywords too.
```suggestion
// JSON array values must be of the string, number, boolean, object, array, or null JSON types.
```